recirq/quantum_chess/quantum_board_test.py

```diff
@@ -4,1030 +4,1220 @@
 # you may not use this file except in compliance with the License.
 # You may obtain a copy of the License at
 #
-#     https://www.apache.org/licenses/LICENSE-2.0
+#     https://www.apache.org/licenses/LICENSE-2.0
 #
 # Unless required by applicable law or agreed to in writing, software
 # distributed under the License is distributed on an "AS IS" BASIS,
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
 import pytest
+import cirq
 
 import recirq.engine_utils as utils
 import recirq.quantum_chess.bit_utils as u
 import recirq.quantum_chess.enums as enums
 import recirq.quantum_chess.move as move
 import recirq.quantum_chess.quantum_board as qb
 from recirq.quantum_chess.test_utils import (
-    assert_sample_distribution,
-    assert_samples_in,
-    assert_this_or_that,
-    assert_prob_about,
-    assert_fifty_fifty,
+    assert_sample_distribution,
```

Comment:
This line has a non-breaking space in it.

I get this error:
`SyntaxError: invalid non-printable character U+00A0`

It can happen if you cut and paste from some HTML.  I would make sure to include only spaces for your whitespace.  Running yapf or black should also clean this up.